src/pages/Scan.vue

```diff
@@ -71,17 +79,20 @@ export default {
             vouchers : Store.trader.vouchers,
             recVouchers : Store.trader.recVouchers,
             errorMessage : Store.error,
+            netMgr : Store.netMgr,
+            queueMessage : false,
             spinner: false,
             validate: false,
-            fail: false
+            fail: false,
+            queued: false,
         }
     },
     watch: {
         voucherCode : function(code) {
             if (code.length === parseInt(this.$refs.voucherBox.getAttribute("maxlength"))) {
                 this.$refs.submitVoucher.click();
             }
-        }
+        },
```

Comment:
Trailing commas are good for a few reasons - but inconsistent use of them negates the goodness. So I was just checking because it looked inconsistent.